.github/workflows/ci.yml

```diff
@@ -85,6 +91,12 @@ jobs:
         run: |
           docker push "$CORE_CI_IMAGE:latest"
           docker push "$CORE_CI_IMAGE:branch--$BRANCH_REF"
+      - name: Push dependabot-core-ci image to GHCR mirror
+        run: |
+          docker tag "$CORE_CI_IMAGE:latest" "$CORE_CI_MIRROR:latest"
+          docker push "$CORE_CI_MIRROR:latest"
+          docker tag "$CORE_CI_IMAGE:latest" "$CORE_CI_MIRROR:branch--$BRANCH_REF"
+          docker push "$CORE_CI_MIRROR:branch--$BRANCH_REF"
```

Comment:
I've left it alone for now, but one thing of note is that I believe pushing `latest` or `branch--$BRANCH_REF` should be mutually exclusive depending on the ref being built.

We should only really be pushing `latest` when building `main` as otherwise the `latest` image is actually juddering between CI builds, meaning it's not clear if each build is caching from `main` or a sibling CI build on another PR. I think this may mean our builds are currently subject to some fuzzing 🤔